setup.py

```diff
@@ -16,7 +16,7 @@
     install_requires=[
         "arviz==0.10.0",
         "astor",
-        "jax==0.1.77",
+        "jax",
```

Comment:
Since JAX is a very young library I think we should still pin the version (to the latest one now), and bump it after having read the release notes. Otherwise we might get a bunch of weird bugs in the future when JAX makes a new release.

Reply:
OK! I just updated this